experimenter/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -1209,8 +1219,8 @@ def test_update_experiment_branches_with_feature_configs(self):
             "description": "a control",
             "ratio": 1,
             "featureValues": [
-                {"featureConfig": feature1.id, "value": ""},
-                {"featureConfig": feature2.id, "value": ""},
+                {"featureConfig": str(feature1.id), "value": ""},
+                {"featureConfig": str(feature2.id), "value": ""},
```

Comment:
Why didn't you use f-strings here? Is ```feature1.id``` an integer?

Reply:
I didn't think to use f-strings here. `feature1.id` is an integer and calling str() seem simple enough.

Reply:
It is only a test, and we're using str() elsewhere for the same thing in that file, so I think I'll leave it be.

Reply:
@jrbenny35 that's interesting about fstrings being faster, but I don't think it'll make a big difference here.  Plus typically fstrings are used for combining some variables with string constants, but here we're just straight up converting a value to a str, and I still think of `str(...)` as the most idiomatic way to do that.